src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+const parent = document.querySelectorAll('.field');
+
+inputs.forEach((item, index) => {
+  const label = document.createElement('label');
+
+  label.classList.add('field-label');
+  label.htmlFor = item.id;
+  label.textContent = item.getAttribute('name');
+
+  parent[index].insertBefore(label, item);
+
+  let placeholder = item.getAttribute('name').split('');
```

Comment:
getAttribute никто не использует. Можно обращаться к любому атрибуту элемента как к свойству обьекта, точно также как ты сделала выше с htmlFor. Не то чтобы это ошибка, просто так намного удобнее
```suggestion
  let placeholder = item.name.split('');
```